source/geometries/Next100FieldCage.cc

```diff
@@ -692,7 +697,20 @@ void Next100FieldCage::BuildELRegion()
 
     new G4PVPlacement(0, G4ThreeVector(0., 0., anode_sim_zpos - grid_thickn_/2.),
                       anode_logic, "ANODE_RING", mother_logic_, false, 0, false);
+
+    /// EL gap.
+    el_gap_solid =
+      new G4Tubs("EL_GAP", 0., gate_ext_diam_/2.,
```

Comment:
Why is the diameter of the el gap changed from `gate_int_diam_` to `gate_ext_diam_` in the case of the SS grids?